Use different amplitudes per channel in `LayoutTests/svg/dynamic-updates/SVGFEComponentTransferElement-dom-amplitude-attr.html`

obyknovenius opened the pull request:
#### 5b3a80e263160410767442939b0b7b498fc36b45
<pre>
Use different amplitudes per channel in `LayoutTests/svg/dynamic-updates/SVGFEComponentTransferElement-dom-amplitude-attr.html`
<a href="https://bugs.webkit.org/show_bug.cgi?id=258211">https://bugs.webkit.org/show_bug.cgi?id=258211</a>

Reviewed by Nikolas Zimmermann.

In <a href="https://commits.webkit.org/265239@main">https://commits.webkit.org/265239@main</a> @nikolaszimmermann suggested
using different amplitudes per channel to make them more distinct in
the output.

* LayoutTests/svg/dynamic-updates/SVGFEComponentTransferElement-dom-amplitude-attr-expected.txt:
* LayoutTests/svg/dynamic-updates/SVGFEComponentTransferElement-dom-amplitude-attr.html:

Canonical link: <a href="https://commits.webkit.org/265348@main">https://commits.webkit.org/265348@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e5e8ddc2c375df46e8ccf9f8162ef66df2a02546

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/10699 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/10919 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/11210 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/12348 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/10257 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/13293 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/10892 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/13170 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/10859 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/11773 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/8993 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/12750 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/9067 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/9652 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/16909 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/10137 "Passed tests") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/9803 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/13053 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/10270 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/8363 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/9429 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/9543 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/2552 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/13702 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/10132 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nikolaszimmermann commented on LayoutTests/svg/dynamic-updates/SVGFEComponentTransferElement-dom-amplitude-attr.html:
```diff
@@ -66,21 +66,21 @@ <h1>SVG 1.1 dynamic update tests</h1>
-	feBFunc.setAttribute("amplitude", "1");
-	feAFunc.setAttribute("amplitude", "1");
+    feRFunc.setAttribute("amplitude", "5");
+	feGFunc.setAttribute("amplitude", "6");
```
I would have used the opportunity to fix the trailing white space as well.

obyknovenius replied:
Done.

nikolaszimmermann reviewed (approved):
r=me. Thanks Vitaly for the prompt patch.

obyknovenius committed:
Use different amplitudes per channel in `LayoutTests/svg/dynamic-updates/SVGFEComponentTransferElement-dom-amplitude-attr.html`
https://bugs.webkit.org/show_bug.cgi?id=258211

Reviewed by Nikolas Zimmermann.

In https://commits.webkit.org/265239@main @nikolaszimmermann suggested
using different amplitudes per channel to make them more distinct in
the output.

* LayoutTests/svg/dynamic-updates/SVGFEComponentTransferElement-dom-amplitude-attr-expected.txt:
* LayoutTests/svg/dynamic-updates/SVGFEComponentTransferElement-dom-amplitude-attr.html:

Canonical link: https://commits.webkit.org/265348@main
